### DISCUSSION TITLE
Share Code from a GitHub Repo with a Permalink

### DISCUSSION THREAD

**Discussion by ghost** [textadventures/quest#1261] [Libraries and Code Samples]
Hello,

Let's say someone has asked a question in Discussions, and you want to share the `UpdateObjectLinks` function's code.

![image](https://github.com/user-attachments/assets/3fc48e77-5ddb-4bc2-97a5-e5a69a2358a5)

---
I rarely want to search the documentation when I'm trying to find the actual code (`:)`), so I always begin my searches' text with `NOT docs`.

![image](https://github.com/user-attachments/assets/10c7119d-10e1-4f60-a7c5-d1d6927efd87)

---
After scrolling a bit to find the code I wish to share, I click on the line number I want to start with. (You can't see my pointer, but you can see `296` is blue in **WorldModel/WorldModel/Core/CoreScopes.aslx**)

![image](https://github.com/user-attachments/assets/2290440a-5efb-4d69-bd4a-85898afc9ae3)

---
Now, you should see something like this:
![image](https://github.com/user-attachments/assets/a6eb453f-693d-4bba-8f1e-abdebd9ec46f)

---
I want to share lines 296-314. Line 296 is already selected, so I shall hold SHIFT and click 314 to highlight the entire block of code (much like a normal text editor).

![image](https://github.com/user-attachments/assets/e4cffd84-9371-410f-9221-ff0b6728d5d1)

---
Now, I can just click the `...` thingy and select 'Copy permalink'
![image](https://github.com/user-attachments/assets/100af9f0-f462-4264-a868-5d7bbbe5a697)

---
When I paste that into my GitHub Discussions post:

https://github.com/textadventures/quest/blob/9e370290d488911f43aae09d0355a05ceeafef90/WorldModel/WorldModel/Core/CoreScopes.aslx#L296-L314

**Comment by KVonGit** [comment]
..and I see that it does not show the code in the mobile app. `:(`
